translate/translate.go

```diff
@@ -79,20 +100,22 @@ func ToLingvanexRequest(r *http.Request, serverURL string) (*http.Request, bool,
 	}
 	qVals := r.PostForm["q"]
 
-	lnx_to, err := language.ToLnxLanguageCode(to)
+	charsProcessed.Add(float64(len(qVals)))
```

Comment:
@evq 
Does it work well if the request has multiple `q` fields?
![image](https://user-images.githubusercontent.com/45488748/204814525-19b6f8e3-1933-43f0-a49d-644929312d07.png)